launch/lifelong_launch.py

```diff
@@ -1,17 +1,71 @@
-from launch import LaunchDescription
-import launch_ros.actions
 from ament_index_python.packages import get_package_share_directory
+from launch import LaunchDescription
+from launch.actions import (DeclareLaunchArgument, EmitEvent, LogInfo,
+                            RegisterEventHandler)
+from launch.conditions import IfCondition
+from launch.events import matches_action
+from launch.substitutions import (AndSubstitution, LaunchConfiguration,
+                                  NotSubstitution)
+from launch_ros.actions import LifecycleNode
+from launch_ros.event_handlers import OnStateTransition
+from launch_ros.events.lifecycle import ChangeState
+from lifecycle_msgs.msg import Transition
 
 
 def generate_launch_description():
-    return LaunchDescription([
-        launch_ros.actions.Node(
+    autostart = LaunchConfiguration('autostart')
+    use_lifecycle_manager = LaunchConfiguration("use_lifecycle_manager")
+
+    declare_autostart_cmd = DeclareLaunchArgument(
+        'autostart', default_value='true',
+        description='Automatically startup the slamtoolbox. '
+                    'Ignored when use_lifecycle_manager is true.')
+    declare_use_lifecycle_manager = DeclareLaunchArgument(
+        'use_lifecycle_manager', default_value='false',
+        description='Enable bond connection duaring node activation')
```

Comment:
```suggestion
        description='Enable bond connection during node activation')
```

Reply:
Fix for all launch files

Reply:
Sorry. fixed [e832151532292d39b63c16fa7e8f5a46847e77bf]